integration-tests/integration/activity/activityList.spec.js

```diff
@@ -37,7 +37,6 @@ context('Activity list page', () => {
     offenders.forEach((offender) => {
       cy.task('stubOffenderBasicDetails', offender)
     })
-    cy.task('stubGetAbsenceReasons')
```

Comment:
don't need to call the absence reasons version now as have switched to v2 for all